sympy/ntheory/primetest.py

```diff
@@ -621,6 +624,10 @@ def isprime(n):
         return mr(n, [2, 123635709730000, 9233062284813009, 43835965440333360, 761179012939631437, 1263739024124850375])
     if n < 18446744073709551616:
         return mr(n, [2, 325, 9375, 28178, 450775, 9780504, 1795265022])
+    if n < 318665857834031151167461:
```

Comment:
These look suspiciously small relative to the numbers in the other ranges.

Reply:
https://arxiv.org/pdf/1509.00864.pdf